### ISSUE TITLE
Unclear documentation and VaultInvalidConfigError

### ISSUE THREAD

**Issue by alzaki-clgx** [SaiHemanthBR/vault-api#15]
The Wiki does not provide a basic example to answer the following:

"I have VAULT_TOKEN, VAULT_ADDR, and I provide the `path` in the following code:
```
const Value = await vault({
      method: 'read',
      path: '.../.../.../KEY'
    })
```
These three things should be enough to retrieve a value from my vault's kv pair, ergo what is an example of the simplest config that need only these 3 things to make my app not land me in the same puddle as this person: https://stackoverflow.com/questions/76114726/vaultinvalidconfigerror-how-can-i-add-a-simple-config-to-typescriptss-vault-ap"

Secondly, I read the docs and realize that I should first create a `vault-api` instance
<img width="731" alt="Screenshot 2023-04-26 at 1 35 52 PM" src="https://user-images.githubusercontent.com/64810019/234696267-ab7efe01-4124-401d-8936-481c9ee98d0a.png">

However as seen above my 2023.1 IntelliJ with Version 4.9.5 TypeScript doesn't know what `create` is

**Comment by mjeghers-tmx** [comment]
Yes this documentation is very sparse, and I am left with no idea how to make a simple "Hello World"-level example program work.  The code snippers given don't work and I don't know what else is needed.

Please document better!

**Comment by alzaki-clgx** [comment]
I emailed SBR but no response yet, I am compelled to open a Pull Request but until then I'm going to use https://stackoverflow.com/questions/65702556/vault-hashicorp-curl-equivalent-of-vault-read

**Comment by alzaki-clgx** [comment]
@bshreddy is back online!
